### ISSUE TITLE
Popover menu items should receive keyboard focus and interaction

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/sul-embed#1727]
### Expected behavior

The Share and Download items in the popover menu should receive keyboard focus, and be "clickable" via keyboard Return.

### Current behavior

The popover menu dialog receives focus (and can be dismissed with Escape), but the Share and Download items do not receive focus, and cannot be opened.

### Steps to reproduce

Use Tab/Shift+Tab and Return keys to navigate through the viewer.

### Suggested approach

Add `tabindex="0"` to the `<li>` elements to make them focusable. NB: this does not also make the items "clickable" via keyboard Return. It may be easier to use `<button>` elements here instead.
